packages/dds/tree/src/feature-libraries/default-schema/defaultEditBuilder.ts

```diff
@@ -167,9 +171,15 @@ export class DefaultEditBuilder implements ChangeFamilyEditor, IDefaultEditBuild
 
 	public constructor(
 		family: ChangeFamily<ChangeFamilyEditor, DefaultChangeset>,
-		changeReceiver: (change: DefaultChangeset) => void,
+		private readonly mintRevisionTag: () => RevisionTag,
+		changeReceiver: (change: TaggedChange<DefaultChangeset>) => void,
 	) {
-		this.modularBuilder = new ModularEditBuilder(family, fieldKinds, changeReceiver);
+		this.modularBuilder = new ModularEditBuilder(
+			family,
+			fieldKinds,
+			mintRevisionTag,
```

Comment:
Nit: it's a tinsy bit ugly how DefaultEditBuilder and ModularEditBuilder both end up with a mintRevisionTag field that they both use. It would be nicer if only one of them were responsible for minting new revisions. It seems like it should be easy to fix that by having `ModularEditBuilder.addNodeExistsConstraint` take in an extra argument for the revision tag.

Reply:
Yes, good point. I will update it to mint revision tags from one place.